src/main/java/com/stream_pi/client/window/dashboard/actiongridpane/ActionBox.java

```diff
@@ -474,7 +481,48 @@ public void setDisplayTextFontColourAndSize(String colour)
             displayTextLabel.setStyle(totalStyle);
         }
     }
-
+    
+    public void playActionAnimation() throws SevereException
+    {
+        Config config = Config.getInstance();
+        switch(config.getCurrentAnimationName())
+        {
+            case "None":
+                break;
+            case "Flip":
+                new Flip(getChildren().get(1)).play();
+                break;
+            case "Bounce":
+                new Bounce(getChildren().get(1)).play();
+                break;
+            case "Jack In The Box":
+                new JackInTheBox(getChildren().get(1)).play();
+                break;
+            case "Swing":
+                new Swing(getChildren().get(1)).play();
+                break;
+            case "Jello":
+                new Jello(getChildren().get(1)).play();
+                break;
+            case "Pulse":
+                new Pulse(getChildren().get(1)).play();
+                break;
+            case "RubberBand":
+                new RubberBand(getChildren().get(1)).play();
+                break;
+            case "Shake":
+                new Shake(getChildren().get(1)).play();
+                break;
+            case "Tada":
+                new Tada(getChildren().get(1)).play();
+                break;
+            case "Wobble":
+                new Wobble(getChildren().get(1)).play();
+                break;
+            default:
+                Logger.getLogger("").warning("Invalid Option/n Please contact quimodotcom to solve this error!");
```

Comment:
better idea would be to throw a minor exception and report to user. Also, the error message should be something like 

> Invalid Animation Type

The `Please contact ...` is redundant.

Reply:
done